projects/2019_12_16_MultiSig_QT_Wallet.md

```diff
@@ -107,3 +107,35 @@ The primary objective for this milestone is to provide a way to send and receive
 12. Users can receive transactions from any other address on the network. 
 13. A DMG file is available as part of the PKT Cash Wallet releases [here](https://github.com/artrepreneur/PKT-Cash-Wallet/releases)
 
+## Milestone 2 (M2) report - 2020-05-20 ##
+The two overarching objectives of this phase were to get multisig transactions enabled and additional wallet tools functional. 
+
+### Project Results (Required) ###
+1. Send transactions to multiple addresses at once.
+* Though the GUI has this capability, it was disabled for this version as the behavior of ``sendmany`` rpc is inconsistent at this time.  
+2. Batch transactions for large UTXO input sets.
+* A fold option was added to the menu to fold large UTXO sets into smaller ones, eliminating the necessity for batching.
+3. Sign transactions.
+* All transactions are signed. Of particular interest is the signing of multsig transactions which is possible. 
+* Multisig transactions can be generated, shared with signatories (using a generated QR code) and sent once the minimum number of signatories is acheived.
+4. Verify transactions.
+* Transactions can be imported, decoded and verified. 
+5. Generate new regular addresses and new multi-sig addresses.
+* Multsig address and regular addresses can be generated.
+6. Export private keys and import new private keys.
+* All private keys can be exported and imported.
+7. Encrypt and decrypt messages.
+* This feature has been deemed not necessary at this time.
+8. Password-protect the wallet.
+* The wallet can be protected with a passphrase during setup, or at any time in the future.
```

Comment:
1) I couldn't figure out how to delay the passphrase  until after setup
2) It would be nice to have a "view passphrase" option to make sure I'm typing what I think I'm typing
3) It would be good to let users know in advance what the passphrase requirements are, for instance "1 or more numbers and 1 or more uppercase letters"

Reply:
I have added this feature and it is avail in version beta.27